conduct.md

```diff
@@ -52,35 +52,35 @@ TODO: Write an introductory paragraph.
 
 ### Communication
 
-TODO: Write an introductory paragraph.
+Communication is key to both successful and sustainable code. However it's important that all communication between students and team members follows the three pillars of Respect, Trust, and Humility. In general, students will *respect* their peers, *trust* their contributions and thoughts, and practice *humility* in all aspects of communication, in person, on GitHub, in the Discord, and otherwise.
 
 #### Acceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Team members communitcating in a professional and respectful manner. 
+- Team members should use words like "Please" and "Thank you".
+- Team members will allow other members and teams speak freely. 
 
 #### Unacceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Team members communicate in a way which offends or disrespects others. 
+- Team members constantly interrupting others - we all have something to share. 
+- Team members show signs of passive aggresive behaivor. Working as a team means keeping emotions out of work.
 
 ### Participation
 
-TODO: Write an introductory paragraph.
+Participation for students will be a multidide of factors. We all have bad days and  we all have topics we know more or less about but how many times a student speaks a day will not soley make up their participation. It is being prepared each day, having done the homework, coming to class with an open mind to learn and making an effort to be apart of either small group or large group conversations that will reflect students participation.
 
 #### Acceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Students will respect all peers opinions.
```

Comment:
In the previous category `Team members` was used. However, `Students` is used here. Is there a difference between the two, or are they referring to the same people. I think it's important to clarify that.